tools/internal_ci/linux/grpc_run_tests_matrix.sh

```diff
@@ -23,7 +23,13 @@ source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 # If this is a PR using RUN_TESTS_FLAGS var, then add flags to filter tests
 if [ -n "$KOKORO_GITHUB_PULL_REQUEST_NUMBER" ] && [ -n "$RUN_TESTS_FLAGS" ]; then
   sudo apt-get update
-  sudo apt-get install -y jq
+  retry=0
```

Comment:
this is not the only place where jq gets installed (in fact I'm surprised what is this snippet doing in this script).

let's do this unrelated change in a separate PR, it doesn't really fit under the "adding ptrace to asan" umbrella.